source/shared/cpp/ObjectModel/TextBlock.cpp

```diff
@@ -165,3 +168,126 @@ void TextBlock::SetHorizontalAlignment(const HorizontalAlignment value)
     m_hAlignment = value;
 }
 
+std::string TextBlock::ParseISO8601UsingRegex() const
```

Comment:
>ParseISO8601UsingRegex [](start = 23, length = 22)

I tried loading up the TextBlock.DateTimeFormatting.json sample in the UWP visualizer, and it looks like the date isn't working...

Reply:
it's because spacing in regex; it will update new commits.

---
In reply to: [149149659](https://github.com/Microsoft/AdaptiveCards/pull/871#discussion_r149149659) [](ancestors = 149149659)